packages/e2e-test-app/jest.setup.js

```diff
@@ -26,3 +27,5 @@ global.jasmine.getEnv().addReporter({
     }
   },
 });
+
+LogBox.ignoreAllLogs(true);
```

Comment:
This can maybe be removed. Console feeds to Jest, and console feeds to logbox, but I need to check how stub Logbox feeds to Jest.